### PR TITLE
Add __FreeBSD__ where needed to properly compile on FreeBSD

### PR DESCRIPTION
**Pull request by DianeBruce**
Compiles and runs fine on FreeBSD with these minor changes.

### CODE REVIEW

**Comment by Dantali0n** [comment]
Wow that's pretty neat and which such minimal changes to. :+1:

**Comment by Mur77** [comment]
Hello!
I have successfully build it on FreeBSD, but I have no sound output. I've tried to make with -DUSE_AUDIO_PULSE=1 and -DUSE_AUDIO_ALSA=1, but still nothing.

How did you managed to get sound?
Thanks!

**Comment by DianeBruce** [comment]
> Hello!
> I have successfully build it on FreeBSD, but I have no sound output. I've tried to make with -DUSE_AUDIO_PULSE=1 and -DUSE_AUDIO_ALSA=1, but still nothing.
> 
> How did you managed to get sound?
> ThanksI

If you installed with pkg install cubicsdr it would have installed pulseaudio as well. You need pulseaudio for sound. Hope this helps!

File: src/CubicSDRDefs.h [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: GPL-2.0+
 
 #pragma once
+#include <sys/param.h>
```
Comment (vsonnier) [inline src/CubicSDRDefs.h]: This one is not portable, meaning doesn't work under Windows. Please put it differently to only apply to Linux and friends.

**Comment by vsonnier** [comment]
Nevermind, I've cherry-picked the change in `master`, adapting it for Windows.

**Comment by DianeBruce** [comment]
Apologies! I was busied out elsewhere. Thanks for fixing this!

**Pull request closed without merging**